tests/e2e/runner/runner.go

```diff
@@ -136,20 +140,23 @@ done:
 	err := ctx.Err()
 	if err != nil {
 		cancel()
-		return "", "", err
+		return "", "", 0, err
 	}
 	cancel()
 
 	if blockchainID == "" {
-		return "", "", errors.New("BlockchainId not found")
+		return "", "", 0, errors.New("BlockchainId not found")
 	}
 	if logsDir == "" {
-		return "", "", errors.New("logsDir not found")
+		return "", "", 0, errors.New("logsDir not found")
+	}
+	if pid == 0 {
+		return "", "", pid, errors.New("pid not found")
 	}
-	return blockchainID, logsDir, nil
+	return blockchainID, logsDir, pid, nil
 }
 
-func GetClusterInfo(blockchainId string, logsDir string) (clusterInfo, error) {
+func GetClusterInfo(blockchainId string, logsDir string, pid int) (clusterInfo, error) {
```

Comment:
We should not need to pass a process ID into a getter function to return in the output.

Reply:
both logsDir and pid are passed to this probably ill-named function that also ask server for uris and save cluster info data (including given logsDir+pid) to disk before returning it. Renaming to SaveClusterInfo